wagtail/documents/rich_text.py

```diff
@@ -62,10 +64,15 @@ class DocumentLinkElementHandler(LinkElementHandler):
     def get_attribute_data(self, attrs):
         Document = get_document_model()
         try:
-            doc = Document.objects.get(id=attrs['id'])
-        except Document.DoesNotExist:
+            id = int(attrs['id'])
```

Comment:
In general, we should not cast PKs to `int` as it prevent the user from using UUIDs.

I understand that the `Document` model is built-in and therefore cannot use UUIDs, therefore it's fine here, but Wagtail makes such assumptions in different places and it has been a pain point in the past.